### ISSUE TITLE
Uploading PGX does not reload pgx table

### ISSUE THREAD

**Issue by ivokwee** [bigomics/omicsplayground#208]
After uploading a PGX file via the upload tab, the main data table is not triggered for update and menu is also not appearing. New file is not appearing in table.

**Comment by mauromiguelm** [comment]
I am trying to reproduce the error, so I tried a few tests on latest v3 master (ebfbce6) and here is the following:

- When I upload the samples, contrast, counts via upload, I get the error:
```
Matching gene set matrix...
Reducing gene set matrix...
Warning: Error in order: argument 1 is not a vector
  82: h
  81: .handleSimpleError
  80: order
```
Has anyone seen this or its just me?

- I cannot get to the v3 deployment testing as login is not working in any browser for me (chrome, edge and firefox).

I can try to debug on the same version as deployment, but I guess its better if we solve this on latest master, right?

**Comment by ivokwee** [comment]
I think the 'ngs' not found is a different problem. This original issue is just that the PGX table is not triggered to be renewed so the uploaded PGX is not visible in the table and also does not trigger to show the menu and go to dataview by default.

**Comment by mauromiguelm** [comment]
**fixed:** Warning: Error in order: argument 1 is not a vector

**working on second issue:** Warning: Error in observe: object 'ngs' not found

**Comment by mauromiguelm** [comment]
fixed, see c91ae0e91
